plugins/examples/dynamic-clusters/src/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Headlamp, Plugin, Registry } from '@kinvolk/headlamp-plugin/lib';
+import { Button } from '@material-ui/core';
+import React from 'react';
+
+function ClusterCreationButton() {
+  return (
+    <Button
+      onClick={() => {
+        Headlamp.setCluster({
+          name: 'my-plugin-set-cluster',
+          server: 'http://phony.abcde-example-k8s-cluster.k8scluster',
+        }).then(() => {
+          // We should find a better way to make sure the clusters info is reloaded.
```

Comment:
I think window.location.reload is good enough, so probably the comment can be removed?